gcs/tailwind.config.js

```diff
@@ -15,7 +15,9 @@ export default {
           DEFAULT: '#E74C3C',
         },
         falcongrey: {
+          120: '#171A1B',
```

Comment:
I don't think we should go above 100 otherwise it doesn't make sense. Does `falcongrey-100` not work?

Reply:
i mean we could but its a darker colour than 100 and so we'd have to change _all_ instance of falcongrey-100 which could be annoying. Tailwind goes to 950 so we should actually be using those values (100, 200, 300, 400, 500, 600, 700, 800, 900, 950)

Reply:
What if you use a tailwind gray instead of a falcongrey?